extensions/amp-story/1.0/amp-story-bookend.css

```diff
@@ -268,17 +285,8 @@
 }
 
 .i-amphtml-story-bookend-component-set {
+  margin: 0 32px !important;
```

Comment:
Can we do `24px` to match the margin-bottom applied on the bookend components? That'd also match other drawers we have, like `story-consent`.